edgexfoundry/edgex-compose
# build: Update to latest version of Consul

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make run no-secty ds-virtual` from compose builder
verify all service bootstrapped successfully
Turn DEBUG logging on for rules engine via Consul UI
Verify via rules engine logs that data is flow
run `make clean`
run `make run ds-virtual` from compose builder
verify all service started bootstrapped successfully
Turn DEBUG logging on for rules engine via Consul UI (requires ACL token)
Verify via rules engine logs that data is flow
